### PR TITLE
Feature add store page

### LINKED ISSUE #105: Implement flowers store page

### PR DESCRIPTION
**Pull request by conalli**
## Description

Write a brief description about your changes...

## Closes issue(s)

Closes #105

## Screenshots

florist-1

## Changes include

- [x] New feature (non-breaking change that adds functionality)
- [x] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [x] I have tested this code

### CODE REVIEW

File: src/pages/Florist/FloristWithCart/FloristWithCart.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  );
+  const calledOnMount = useRef(false);
+  useEffect(() => {
+    if (!calledOnMount.current) {
+      getAllFlowers();
+    }
+    calledOnMount.current = true;
+  }, [getAllFlowers]);
+
+  const [cart, setCart] = useState<Flower[]>([]);
```
Comment (conalli) [inline src/pages/Florist/FloristWithCart/FloristWithCart.tsx:82]: This is the previous version for buying multiple flowers at a time with a cart, currently unused.

Comment (daiHash) [reply src/pages/Florist/FloristWithCart/FloristWithCart.tsx:82]: Could you rename the component and add a comment

File: src/store/user/UserStateAtoms.ts [hunk 2]
```diff
@@ -8,7 +7,7 @@ type UserData =
       imageUrl: string | null;
       isLoggedIn: true;
       balance: number | null;
-      flowerCollections: Flower[];
+      flowerCollections: string[];
```
Comment (daiHash) [inline src/store/user/UserStateAtoms.ts:10]: 👍

**Review by daiHash — APPROVED** [review]
Nice LGTM!

**Review by kome12 — APPROVED** [review]
LGTM!

**Pull request merged**